src/clean_air/visualise/render_plot.py

```diff
@@ -4,15 +4,64 @@
 
 import hvplot.xarray  # noqa
 
+from iris.cube import Cube, CubeList
+import iris.plot
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+
 
 class Plot:
-    """
-    Class to process data and create a lovely map from it.
-    """
     def __init__(self, dataframe):
-        self.df = dataframe.load()
+        # Note: This should always be a CubeList, even if it only contains
+        # one Cube.  This is just for simplification of the code.
+        self.dataframe = dataframe
 
     def render_timeseries(self):
-        # NOTE: Must have a a deployed Bokeh Server app or a deployed Panel
-        # app to be able to view these plots.
-        self.df.hvplot.line(datashade=True)
+        # First make a figure, then add all datasets as subplots:
+        fig = plt.figure(figsize=(11, 5))
+
+        # We want 3 plots per row unless there are less than 3 plots to
+        # render, then we want the figsize to match the number of plots:
+        n_plots = len(self.dataframe)
+        n_rows = int(n_plots-1/3) + 1
+        if len(self.dataframe) <= 3:
+            n_cols = len(self.dataframe)
+        else:
+            n_cols = 3
+
+        for i, cube in enumerate(self.dataframe):
+            # First check that cube is 1-dimensional, otherwise reduce
+            # dimensions:
+            if cube.ndim > 1:
+                for coord in cube.dim_coords:
+                    if coord.points.size == 1:
+                        cube = cube.collapsed(coord, iris.analysis.MEAN)
+                    else:
+                        pass
```

Comment:
this bit is redundant, so could be removed

Reply:
Yes, you're not wrong.  It'll make me a little uncomfortable removing it, but I think it's just an OCD thing that I can ignore.  I will remove these lines.